projects/core/directives/dropdown/dropdown.component.ts

```diff
@@ -1,19 +1,18 @@
 import {ChangeDetectionStrategy, Component, computed, inject} from '@angular/core';
 import {takeUntilDestroyed} from '@angular/core/rxjs-interop';
+import {TuiAnimated} from '@taiga-ui/cdk';
```

Comment:
Eslint did not autofix it 🤔 

```suggestion
import {TuiAnimated} from '@taiga-ui/cdk/directives/animated';
```